src/pages/public_registration/action_accordions/PublishingAccordion.tsx

```diff
@@ -190,6 +212,8 @@ export const PublishingAccordion = ({
   const unpublishedOrDeleted =
     registration.status === RegistrationStatus.Deleted || registration.status === RegistrationStatus.Unpublished;
 
+  const published = registration.status === RegistrationStatus.Published;
```

Comment:
Ser at variabelen over også mangler "is"

```suggestion
  const isPublished = registration.status === RegistrationStatus.Published;
```